### ISSUE TITLE
Images with .stretch class are not visible in overview mode

### ISSUE THREAD

**Issue by sanand0** [hakimel/reveal.js#1187]
In reveal.js 3.0.0, the following slide markup:

```
<section>
  <img class="stretch" data-src="https://s3.amazonaws.com/hakim-static/reveal-js/arrow.png">
</section>
```

when rendered in overview mode, reduces to a 1px x 1px image:

![image](https://cloud.githubusercontent.com/assets/63734/6965505/a6d42a12-d96f-11e4-8513-02c912f4a04b.png)

It renders fine as a regular slide, though.

**Comment by CtrlC-Root** [comment]
I can confirm this with release `3.6.0`.

**Comment by hakimel** [comment]
Fixed in dev via #1188, will be released with 3.8.0
